frontend/kunchikal-falls/kunchikal-falls.css

```
@@ -0,0 +1,2944 @@
+}
+
+.menu-toggle .bar {
+  display: block;
+
+  width: 21px;
+
+  height: 2px;
+
+  margin: 4px auto;
+
+  background: currentColor;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)